cmd/localkube/cmd/start.go

```diff
@@ -79,6 +79,12 @@ func SetupServer(s *localkube.LocalkubeServer) {
 			panic(err)
 		}
 	}
+	if s.ShouldGenerateKubeconfig {
+		if err := s.GenerateKubeconfig(); err != nil {
+			fmt.Println("Failed to create kubeconfig!")
+			panic(err)
```

Comment:
handle this error by logging it to glog and exiting

Reply:
sorry I didn't see this.  ShouldGenerateCerts panics also.  We should add a PR to change both of them.

Reply:
sorry I didn't see this.  ShouldGenerateCerts panics also.  We should add a PR to change both of them.